pkg/test/framework/components/istio/ingress.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	getAddressTimeout = retry.Timeout(3 * time.Minute)
+	getAddressTimeout = retry.Timeout(6 * time.Minute)
```

Comment:
@luksa you can revert this change. Tests will faill until we merge this: https://github.com/maistra/istio/pull/761/commits/aede273d44a22db2dc630bb3770dbacaf7df80c9.

Reply:
3 minutes seems a bit low. I think I remember LB provisioning taking up to 5 minutes.

Reply:
LB was failing, because of missing image, so this change is no longer needed.

Reply:
@yannuil can you remove this commit?